tcms/testplans/templates/testplans/get.html

```diff
@@ -153,6 +153,10 @@ <h2 class="card-pf-title">
                             <span class="fa fa-search"></span>
                             <span class="js-test-case-tester"></span>
                         </div>
+                        <div title="{% trans 'Automated' %}" class="list-view-pf-additional-info-item">
+                            <span class="fa fa-cog fa-thumbs-up"></span>{# one of the classes is removed by the js #}
+                            <span class="js-test-case-automated"></span>
+                        </div>
```

Comment:
Move the entire div before priority b/c it will affect the visual presentation less. Other icons/elements following this can have varying length which distorts the vertical alignment of the page. This one will be only an icon and the 2 icons are fairly consistent in width.